isso/ext/notifications.py

```diff
@@ -31,6 +33,38 @@
     from _thread import start_new_thread
 
 
+def _format(thread, comment, general_host, key):
+    rv = io.StringIO()
+
+    author = comment["author"] or "Anonymous"
+    if comment["email"]:
+        author += " <%s>" % comment["email"]
+
+    rv.write(author + " wrote:\n")
```

Comment:
Use string formatting instead of concatenation. (it applies for all the function below :) )
Also calling only once `rv.write` for subsequent calls would be better.